src/types/Todo.ts

```diff
@@ -0,0 +1,6 @@
+export interface Todo {
+  id: number;
+  userId: number;
```

Comment:
Given the task description, it seems unnecessary to have a 'userId' field in the Todo interface. Consider removing it if not needed.